### DISCUSSION TITLE
Welcome to umurmur Discussions!

### DISCUSSION THREAD

**Discussion by fatbob313** [umurmur/umurmur#162] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by egrain** [comment]
Hi. 
I'm wondering what this admin functionality is and how I get it. I set the admin_password and tried connecting as "superuser", which was the only way I found which let me enter a password, but it doesn't seem to make a difference. I tried login in as "admin", but there I don't know how to put in a password. I don't even want that much. Just the ability to move clients into channels, because the people I play stuff with are a lazy bunch. I also don't mind if I can only use the name admin or superuser. Just the moving part, that's it. 

Also I have no idea what this means: 
```
# username and groupname for privilege dropping.
# Will attempt to switch user if set. 
# username = "";
# If groupname not set the user's default login group will be used
# groupname = "";
```
Could anyone clarify?

Thanks in advance.

**Reply by egrain** [reply]
Found the solution:
Set admin_password in the umurmur.conf file.
Connect to the server
Go to Access Tokens, click "add" and type in the admin password you've set.
reconnect and BOOM!

Still wouldn't mind anyone explaining the 
username = "";
groupname = "";
lines to me.

Sorry about the huge and bold font in my original post. Didn't know pasted config lines would come out like that.

**Reply by troxor** [reply]
`username` and `groupname` set the non-root UNIX user for the daemon to drop privileges to. Traditionally, the daemon is started as root to read certificates and other privileged operations but "reassigns itself" to a normal user afterward.

**Reply by egrain** [reply]
Totally forgot about this. Thanks.
